src/Components/Publishing/Icon/IconPlus.tsx

```diff
@@ -33,7 +33,11 @@ const Icon: React.SFC<Props> = props => (
   </svg>
 )
 
-export const IconPlus = styled(Icon)`
+export const IconPlus = styled(Icon).attrs<{
+  suppressClassNameWarning?: boolean
+}>({
+  suppressClassNameWarning: true,
```

Comment:
Warnings from styled-components about wrapping non-styled components without passing in the `className` prop (great new warning)

Reply:
Basically this icon isn't doing anything atm because the classname isn't being passed in (it's configured incorrectly). I didn't write this code so this was the easiest way to keep it as it was while preserving old behavior. I'm gonna go back through and cleanup these warnings once i bump to latest SC version (right now we're at 4.0.3 because of some type errors)